cfme/tests/containers/test_node.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from cfme.containers.provider import ContainersProvider
+from cfme.exceptions import CandidateNotFound
+from cfme.utils.log import logger
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+pytestmark = [pytest.mark.usefixtures('setup_provider')]
+
+
+TEST_VIEWS = ('All', 'Details')
+
+
+@pytest.yield_fixture(scope="function")
+def node(provider, appliance):
+    collection = appliance.collections.nodes
+    return collection.instantiate(provider.data['hosts'][0]['name'], provider=provider)
+
+
+@pytest.mark.polarion('CMP-10782')
+@pytest.mark.tier(3)
+@pytest.mark.provider([ContainersProvider])
+def test_navigate_all(node):
+    for test_item in TEST_VIEWS:
+        try:
+            view = navigate_to(node, test_item)
+        except CandidateNotFound:
+            logger.warning(
+                'Exception while attempting to navigate to page')
```

Comment:
**REQUIRED/QUESTION**: Shouldn't this be a test failure as well? Eg. using soft_assert?

Reply:
@mfalesni Yes, it should be. Let me fix that.

Reply:
:+1: done